sys-fs/f2fs-tools/f2fs-tools-1.7.0.ebuild

```diff
@@ -15,7 +13,7 @@ SLOT="0/1"
 KEYWORDS="~amd64 ~arm ~mips ~x86"
 IUSE=""
 
-DEPEND="
+RDEPEND="
 	sys-apps/util-linux
 	sys-libs/libselinux"
```

Comment:
ALso, given that you said it links against the deps, you STILL need to incldue them in `DEPEND`